core/src/main/java/com/scalar/db/storage/jdbc/JdbcUtils.java

```diff
@@ -92,6 +92,10 @@ public static BasicDataSource initDataSourceForTableMetadata(
     dataSource.setMinIdle(config.getTableMetadataConnectionPoolMinIdle());
     dataSource.setMaxIdle(config.getTableMetadataConnectionPoolMaxIdle());
     dataSource.setMaxTotal(config.getTableMetadataConnectionPoolMaxTotal());
+    for (Entry<String, String> entry : rdbEngine.getConnectionProperties().entrySet()) {
+      dataSource.addConnectionProperty(entry.getKey(), entry.getValue());
+    }
```

Comment:
Now we add custom JDBC connection properties if set in the RdbEngine for all JDBC data source used by: 
- `DistributedStorage` => added previously
- `DistributedStorageAdmin` => newly added
- `TableMetadataManager` => newly added

This is motivated by the fact that the Db2 driver requires a configuration property `retrieveMessagesFromServerOnGetMessage` to print a verbose error message that does not contain only the error code.